Calculation.py

```diff
@@ -0,0 +1,18 @@
+import output as op
+import spouseOrParentShares as sps
```

Comment:
Don't abbreviate things with shorthanded terminology, if you ever need that they should be comprehensive and self-explanatory.